simple_history/models.py

```diff
@@ -31,6 +31,13 @@ def default_get_user(request, **kwargs):
         return None
 
 
+def get_remote_addr(request):
+    x_forwarded_for = request.META.get('HTTP_X_FORWARDED_FOR')
+    if x_forwarded_for:
+        return x_forwarded_for.split(',')[0]
```

Comment:
Should we also log the remainder of the chain and the IP of the host that sent the request to Django given that it is easier to spoof IPs at the beginning of the chain?

Reply:
Haven't thought about that, it might be a good idea! Especially in my use case. I'm just not sure how to do it without over-complicating things for the average user. After all, 99% of the time all you want is the first address in the chain.